equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm7Mapper.java

```diff
@@ -308,4 +304,14 @@ public interface Crm7Mapper extends CrmMapper {
     @Mapping(target="name", source="solicitor_sign_name")
     @Mapping(target="date", source="solicitor_sign_date")
     Crm7CaseInformationSolicitorDTO getCaseInformationSolicitorDTOFromModel(Crm7DetailsModel model);
+    @Mapping(target="qualityControl", source="model")
+    @Mapping(target="authority", source="model")
+    Crm7OfficialUseDTO getOfficialUseDTOFromModel(Crm7DetailsModel model);
+    @Mapping(target="decision", source="decision_original")
+    @Mapping(target="decisionReason", source="full_grant_notes")
+    @Mapping(target="destructionDate", source="destruction_date")
+    Crm7DecisionDTO getDecisionDTOFromModel(Crm7DetailsModel model);
```

Comment:
I would change the name of the function to something like getQualityControlDTOFromModel ... so it is clear it will map to the QC section. Otherwise, it is a dodgy mapping from decision --> quality-control

Reply:
Done